pkg/cmd/repo/rename/rename.go

```diff
@@ -170,3 +180,16 @@ func updateRemote(repo ghrepo.Interface, renamed ghrepo.Interface, opts *RenameO
 
 	return remote, err
 }
+
+func renameRepoDir(newRepoName string, opts *RenameOptions) (err error) {
```

Comment:
I'd love to see some tests for this. I wonder if we can cover the different os use-cases in those tests 🤔

Reply:
Should these tests need to be done now? Since renaming the local folder isn't working yet, I think we can write the tests after the feature is implemented

Reply:
Ah, that's up to you. I'm not trying to prescribe the order in which you implement the solution for this, but instead highlighting that we'll want test coverage before this ships.

FWIW, though, I think writing the tests might inform how best to finish the feature, though that's my general workflow as is. I tend to lean on TDD when I'm trying to implement something with some unknown unknowns, though I know that's not how many folks like to work 🙂